library/build.gradle

```diff
@@ -49,11 +49,10 @@ android {
 }
 
 dependencies {
-    implementation "androidx.core:core-ktx:1.0.0-rc01"
-    implementation "androidx.appcompat:appcompat:1.0.0-rc01"
-    implementation "androidx.recyclerview:recyclerview:1.0.0-rc01"
-    implementation "androidx.coordinatorlayout:coordinatorlayout:1.0.0-rc01"
-    implementation "com.google.android.material:material:1.0.0-rc01"
+    api "androidx.appcompat:appcompat:${ANDROID_X_VERSION}"
```

Comment:
Is there a problem leaving these as implementation? Exposing those to other modules may introduce issues, I am not quite sure. It is just a weird in-between time with Jetpack, right now.

Reply:
Should we leave `api "androidx.appcompat:appcompat:${ANDROID_X_VERSION}"` because it is main dependency.

Reply:
Right, it is the main dependency for this project, but there is little reason to leak that information to other projects, if, for whatever reason, they aren't using the support library.

I always go with `implementation`, when I can, especially for library projects. Then, whoever is using this library doesn't have to know or care about the implementation details.

Reply:
Agree. I change them to `implementation`.